README.md

```diff
@@ -12,18 +12,18 @@ A really simple server has been implemented with Express. It offers the minimum
 
 However this server is not perfect. It could be improved and tested as well.
 
-So your mission is to develop the front-end from scratch using a famous front-end framework.
+So your mission is to develop the front-end from scratch using a front-end framework.
 
 We are also expecting from you a usable, responsive UI.
 
 # Process
 
-Fork the repository into your account. Once your code is ready open a pull-request on this repository and we will review it.
+Fork the repository into your account. Once your code is ready, send a pull-request to this repository and we will review it.
 
 # Requirements
 
-* React 14+
-* Redux or Flux or Frint or whateverelse with a one-way data flow
+* React 15+
+* Redux or Flux or [FrintJS](https://frint.js.org) or whatever else with a unidirectional data flow
```

Comment:
Can we replace the usage of "whatever else with a" with "or other alternatives that implement a" ?